tests/test_rootfs.py

```diff
@@ -116,14 +116,32 @@ def test_expected_files_ext234(
                     ]
                 )
 
+                # Check whether DBus files exist
+                output = subprocess.check_output(
+                    [
+                        "debugfs",
+                        "-R",
+                        "ls -l -p /usr/share/dbus-1/system.d",
+                        latest_rootfs,
+                    ],
+                    cwd=tmpdir,
+                ).decode()
+                assert any(
+                    [
+                        line.split("/")[5] == "io.mender.conf"
```

Comment:
This needs to be updated because of https://github.com/mendersoftware/mender/pull/658

Reply:
Ah, good catch, fixed!